modules/core/core-project/src/main/java/com/enonic/xp/core/impl/project/ProjectServiceImpl.java

```diff
@@ -184,7 +224,9 @@ private Project doInitRootNodes( final CreateProjectParams params )
             .build()
             .initialize();
 
-        return Project.from( repositoryService.get( params.getName().getRepoId() ) );
+//        nodeService.refresh( RefreshMode.ALL );
+
+        return Project.from( repositoryService.get( params.getName().getRepoId() ), getContentRootData( params.getName() ) );
```

Comment:
It is problematic to fetch data right after saving. Maybe there is a way to avoid fetch?